packages/media-download/src/yt-dlp.ts

```diff
@@ -17,7 +17,7 @@ const extractInfoJson = (infoJsonPath: string): MediaMetadata => {
 		title: json.title,
 		extension: json.ext,
 		filename: json.filename,
-		mediaPath: `${json.filename}.${json.ext}`,
+		mediaPath: `${json.filename}`,
```

Comment:
the .${json.ext} is not longer needed here as 'ext' always gets included in the filename thanks to the output template